### PR TITLE
Updating Python Run options to reference Smart Send

### PR DESCRIPTION
**Pull request by cwebster-99**
Updating Run options to include Smart Send

### CODE REVIEW

File: docs/python/python-tutorial.md [hunk 1]
```diff
@@ -142,23 +142,23 @@ For full details on editing, formatting, and refactoring, see [Editing code](/do
 
 ## Run Python code
 
-Click the **Run Python File in Terminal** play button in the top-right side of the editor.
+Click the **Run Python File** play button in the top-right side of the editor.
```
Comment (anthonykim1) [inline docs/python/python-tutorial.md:145]: Shouldnt we still have the mentioning of Run Python File in Terminal?

File: docs/python/python-tutorial.md [hunk 2]
```diff
@@ -142,23 +142,23 @@ For full details on editing, formatting, and refactoring, see [Editing code](/do
...
 
-![Using the Run Python File in Terminal button](images/tutorial/run-python-file-in-terminal-button.png)
+![Using the Run Python File button](images/tutorial/run-python-file-in-terminal-button.png)
```
Comment (anthonykim1) [inline docs/python/python-tutorial.md:147]: same thing here - I think we should just leave the terminal wording when it comes to file related things.

File: docs/python/python-tutorial.md [hunk 3]
```diff
@@ -142,23 +142,23 @@ For full details on editing, formatting, and refactoring, see [Editing code](/do
...
 
 There are three other ways you can run Python code within VS Code:
 
-1. Right-click anywhere in the editor window and select **Run > Python File in Terminal** (which saves the file automatically):
+1. Right-click anywhere in the editor window and select **Run Python > Run Python File in Terminal** (which saves the file automatically):
 
    ![Run Python File in Terminal command in the Python editor](images/tutorial/run-python-file-in-terminal.png)
 
-2. Select one or more lines, then press `kbstyle(Shift+Enter)` or right-click and select **Run Selection/Line in Python Terminal**. This command is convenient for testing just a part of a file.
+2. Select one or more lines, then press `kbstyle(Shift+Enter)` or right-click and select **Run Python > Run Selection/Line in Python Terminal**. Alternatively, you can activate Smart Send using `kbstyle(Shift+Enter)` without a selection and the Python extension will send the smallest runnable block of code near where your cursor is placed to the terminal. This command is convenient for testing just a part of a file.
```
Comment (anthonykim1) [inline docs/python/python-tutorial.md:159]: We could also give a reference to existing setting we have to toggle smart send on/off.
This way, people know how to turn the setting back on if they turned it off accidently/unintentionally.

Comment (cwebster-99) [reply docs/python/python-tutorial.md:159]: Just pushed changes to note the setting and alternative behavior

Comment (anthonykim1) [reply docs/python/python-tutorial.md:159]: awesome, looks good - I just sent a little commit fixing typo.

**Review by anthonykim1 — APPROVED** [review]

**Review by luabud — APPROVED** [review]

**Pull request merged**